tests/lib/assertions/test-snapd-recovery-system-pc-24.json

```diff
@@ -0,0 +1,53 @@
+{
+    "type": "model",
+    "authority-id": "test-snapd",
+    "series": "16",
+    "brand-id": "test-snapd",
+    "model": "my-model",
+    "revision": "1",
+    "architecture": "amd64",
+    "timestamp": "2024-04-24T00:00:00+00:00",
+    "grade": "dangerous",
+    "base": "core24",
+    "serial-authority": ["generic"],
+    "snaps": [
+        {
+            "default-channel": "24/edge",
+            "id": "UqFziVZDHLSyO3TqSWgNBoAdHbLI4dAH",
+            "name": "pc",
+            "type": "gadget"
+        },
+        {
+            "default-channel": "24/edge",
+            "id": "pYVQrBcKmBa0mZ4CCN7ExT6jH8rY1hza",
+            "name": "pc-kernel",
+            "type": "kernel"
+        },
+        {
+            "default-channel": "latest/edge",
+            "id": "amcUKQILKXHHTlmSa7NMdnXSx02dNeeT",
+            "name": "core24",
```

Comment:
This should be id `dwTAh7MZZ01zyriOZErqd1JynQLiOGvM`, here and in other places.

Reply:
nice catch, I'll update that